interview-questions.md

```diff
@@ -4,49 +4,75 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: C-create, R-read, U-update, D-delete
 
-  Researched answer:
+  Researched answer: create, read, update, and delete
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: get post put delete patch 
 
-  Researched answer:
+  Researched answer: 
+    GET: The GET method requests a representation of the specified resource. Requests using GET should only retrieve data.
+    
+    HEAD: The HEAD method asks for a response identical to that of a GET request, but without the response body.
+    
+    POST: The POST method is used to submit an entity to the specified resource, often causing a change in state or side effects on the server.
+    
+    PUT:The PUT method replaces all current representations of the target resource with the request payload.
+
+    DELETE: The DELETE method deletes the specified resource.
+    
+    CONNECT: The CONNECT method establishes a tunnel to the server identified by the target resource.
+
+    OPTIONS: The OPTIONS method is used to describe the communication options for the target resource.
+    TRACE: The TRACE method performs a message loop-back test along the path to the target resource.
+
+    PATCH: The PATCH method is used to apply partial modifications to a resource.
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer:
+  Your answer: <div>
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow